Builds/CMake/RippledCompiler.cmake

```diff
@@ -13,7 +13,6 @@ link_libraries (Ripple::common)
 set_target_properties (common
   PROPERTIES INTERFACE_POSITION_INDEPENDENT_CODE ON)
 set(CMAKE_CXX_EXTENSIONS OFF)
-target_compile_features (common INTERFACE cxx_std_17)
```

Comment:
Why is this line removed?

Reply:
Our code should be compiled with at least C++20, and the choice of standard is made by the toolchain file, not our CMake project files.